sdk/core/Azure.Core/src/Shared/NoBodyResponseOfT.cs

```diff
@@ -16,6 +16,9 @@ public NoBodyResponse(Response response)
             _response = response;
         }
 
+        /// <inheritdoc />
```

Comment:
Please add a comment saying this should not be used for new code.

Reply:
I was going to do that, but I think it would be more effective to tweak the analyzer rule instead.

Reply:
I didn't realize that we did this.

Reply:
Since we've filed an issue and haven't addressed it yet, should we at least add a comment as a safeguard?

Reply:
added